src/coverlet.collector/InProcDataCollection/CoverletInProcDataCollector.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Reflection;
+using Coverlet.Core.Instrumentation;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollector.InProcDataCollector;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.InProcDataCollector;
+
+namespace Coverlet.Collector.InProcDataCollector
```

Comment:
Maybe we should have all classes under `Coverlet.Collector.DataCollection`

![image](https://user-images.githubusercontent.com/7894084/57983580-dcfdb700-7a53-11e9-915e-fe651c7252d0.png)

Reply:
Done.